site/docs/workflows/flutter_package.md

```diff
@@ -121,7 +121,7 @@ jobs:
   build:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/flutter_package.yml@v1
     with:
-      coverage_excludes: '*.g.dart'
+      coverage_excludes: '**/**.g.dart'
```

Comment:
```suggestion
      coverage_excludes: '**/*.g.dart'
```

I think there is a subtle difference and `**/*.g.dart` might hit more scenarios?

Reply:
Yeah, that's rabbit hole I didn't want to fall into 😅

I did look at the implementation and under the hood [minimatch](https://github.com/isaacs/minimatch) is being used.

My guess is that `**/**.g.dart` being a bit more eager and catching more scenarios should be a good thing? My head is at "_every generated file anywhere should be ignored_", but that might be me oversimplifying things.

Happy with either option!

Reply:
I agree that we should use whichever is more eager.

I'm not finding examples of patterns ending in `/**.ending`. I believe `**` is more for matching arbitrary directory depth (so it makes more sense to use it in `**/`, `/**/`, or `/**`)

I believe we should use `**/*.g.dart` here, but not totally sure.

Reply:
Alright, gimme 5, I'll update it.